core/platform/email/mailgun_email_services_test.py

```diff
@@ -16,21 +16,39 @@
 
 """Tests for the Mailgun API wrapper."""
 
+import urllib2
 from core.platform.email import mailgun_email_services
```

Comment:
Add newline above this one, to separate stdlib imports from oppia imports.

Reply:
Done